cedar-policy/src/api.rs

```diff
@@ -1985,7 +1985,7 @@ pub struct PolicyId(ast::PolicyID);
 impl FromStr for PolicyId {
     type Err = ParseErrors;
 
-    /// Create a `PolicyId` from a string. Currently always returns Ok().
+    /// Create a `PolicyId` from a string. Currently always returns `Ok()`.
```

Comment:
Do we want to make the other clippy changes?  This should be the only one that's causing CI to fail. IMO it's mostly a question of what will make cherry-picking other changes easier.